### PR TITLE
Create an admin page for viewing all characters and their owners.

### PR DESCRIPTION
**Pull request by Acidity**
Potential problems: It's not paginated so it may take a while to load with thousands of characters.
No actions available on the admin page at the moment.

### CODE REVIEW

File: brave/core/character/template/list.html [hunk 1]
```diff
@@ -93,10 +93,10 @@
     % if web.user.admin:
     <div id="tabs-wrapper">
         <div class="row-fluid">
-            <a class="span3 tab${' active' if not admin else ''}" href="/key/">
+            <a class="span3 tab${' active' if not admin else ''}" href="/character/">
```
Comment (jboning) [inline brave/core/character/template/list.html:96]: haha what.

Comment (Acidity) [inline brave/core/character/template/list.html:96]: Yeah, it originally sent the user over to the key management page...

File: brave/core/character/template/list.html [hunk 2]
```diff
@@ -112,10 +112,14 @@
             <table class="table table-hover">
                 <thead>
                     <tr>
-                        <th class="span3 sortable name">Name</th>
-                        <th class="span4 sortable corporation"><span class="line"></span>Corporation</th>
-                        <th class="span4 sortable alliance"><span class="line"></span>Alliance</th>
+                        <th class="span2 sortable name">Name</th>
```
Comment (jboning) [inline brave/core/character/template/list.html]: what's up with the changing span classes here?

Comment (Acidity) [inline brave/core/character/template/list.html]: Ahhh, got this mixed up with bootstrap where the max width is 12 units. I'll get that fixed.

**Pull request merged**